ajax/libs/deferer/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "deferer",
+  "version": "1.0.0",
+  "description": "Redirect traffic for matching referering hosts",
+  "author": "Dominic Dagradi <ddagradi@gmail.com>",
+  "repository": "https://github.com/dominic/deferer",
+  "main": "deferer.js",
+  "license": "MIT",
+  "dependencies": {},
+  "npmName": "deferer",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "*/"
```

Comment:
Fair; that's the output of `cdnjs-importer`, but the example in the README is different. Sorry about that.

You can just close this if it's waiting on notoriety though. No point keeping a PR around for that.